theblues/identity_manager.py

```diff
@@ -20,28 +20,25 @@
 class IdentityManager(object):
     """Identity Manager API."""
 
-    def __init__(self, url, idm_user, idm_password, timeout=DEFAULT_TIMEOUT):
+    def __init__(self, url, timeout=DEFAULT_TIMEOUT):
         """Initializer.
 
         @param url The url to the identity manager (IdM) API.
-        @param idm_user The user name for the IdM.
-        @param idm_password The password for the IdM.
         @param timeout How long to wait before timing out a request in seconds;
             a value of None means no timeout.
         """
         self.url = ensure_trailing_slash(url)
-        self.auth = (idm_user, idm_password)
         self.timeout = timeout
 
-    def get_user(self, username):
+    def get_user(self, username, macaroons):
         """Fetch user data.
 
         Raise a ServerError if an error occurs in the request process.
 
         @param username the user's name.
```

Comment:
Can you add the macaroon param here, given how often macaroons trip people up. :smile:

Reply:
Done.